test/toolshed/unix_test.exs

```diff
@@ -12,8 +12,7 @@ defmodule Toolshed.UnixTest do
     String.trim(date_time)
   end
 
-  test "cat/1 reads file content" do
-    assert capture_io(fn -> Unix.cat("test/support/test_file.doc") end) ==
-             "Content of this will be read for test purposes"
```

Comment:
It looks like you deleted the unit test for `cat`.

Reply:
lemme just rectify that right now